src/aosm/README.md

```diff
@@ -40,6 +40,24 @@ https://eng.ms/docs/strategic-missions-and-technologies/strategic-missions-and-t
 
 CLI issues should be tagged and triaged as UX bugs.
 
+## Logging
```

Comment:
This section is great. Personal opinion on reading the whole file:
- Bug reporting and Logging should go after the `nfd and nsd commands section, as I feel like I want to know what the heck this tool is before I want to know how to raise a bug and turn logging on
- I also think the new development section that Jamie's added at the bottom should go in setup.md, setup.md should be renamed to development.md and a link should be put in from this doc.  Would you mind making that change too?

Reply:
will do 👍